### ISSUE TITLE
Support for 2.4GHz video modules

### ISSUE THREAD

**Issue by kolins-cz** [markohoepken/rx5808_pro_osd#7]
Hello,
I just tuned my RX6788 2.4GHz module based on RTC6711 chip for the first time. Chip's SPI register is different from RTC6715 (only 24 bits, MSB first and so). I managed to calculate correct values for it and load them via arduino to module. I plan to clean up my code, share it and integrate it to your project. I really like it's features, but I do not use 5.8GHz for video. Would you participate? 
 As I understand, only a few tables and setChannelModule function needs change.
[rtc6711.zip](https://github.com/markohoepken/rx5808_pro_osd/files/468391/rtc6711.zip)
There is my first working version. Values for other frequencies are not included yet.

**Comment by jonathan2018** [comment]
@kolins-cz  
it work ?

**Comment by kolins-cz** [comment]
> 
> 
> @kolins-cz
> it work ?

Yes it works. That is why I want to implement it to this firmware.

**Comment by jonathan2018** [comment]
> > @kolins-cz
> > it work ?
> 
> Yes it works. That is why I want to implement it to this firmware.

thanks.  So code  completed or build more ?  i will order some VRX to try .
Do you have video flights with vrx 2.4ghz from this project ?

**Comment by OrangeFox1980** [comment]
Good day! Here is a question: How can I switch the RTC6711 to SPI control mode?

**Comment by Spider84** [comment]
> SPI mode 
When pin 7 (SPI_SE) is set at high (3.3V), the chip works as in the SPI mode and the pins 
5(SPIDATA), 5(SPILE) and 6(SPICLK)are used for ‘SPI’ inputs for 3-wire programming 
interface. The BX is “don’t care”.

**Comment by OrangeFox1980** [comment]
Thank you. finally I saw it in the datasheet.16 бер. 2025 р. 05:57 Spider ***@***.***> пише:
  
SPI mode
When pin 7 (SPI_SE) is set at high (3.3V), the chip works as in the SPI mode and the pins
5(SPIDATA), 5(SPILE) and 6(SPICLK)are used for ‘SPI’ inputs for 3-wire programming
interface. The BX is “don’t care”.
—Reply to this email directly, view it on GitHub, or unsubscribe.You are receiving this because you commented.Message ID: ***@***.***>



Spider84 left a comment (markohoepken/rx5808_pro_osd#7)

SPI mode
When pin 7 (SPI_SE) is set at high (3.3V), the chip works as in the SPI mode and the pins
5(SPIDATA), 5(SPILE) and 6(SPICLK)are used for ‘SPI’ inputs for 3-wire programming
interface. The BX is “don’t care”.

**Comment by trymrock** [comment]
I'm having trouble selecting the frequency for the RTC6711. Can you help me?

[RTC6711-Revice 2.4GHz.pdf](https://github.com/user-attachments/files/19399960/RTC6711-Revice.2.4GHz.pdf)

[esp8266_test2.4G.zip](https://github.com/user-attachments/files/19399993/esp8266_test2.4G.zip)

**Comment by OrangeFox1980** [comment]
in one day I will look.22 бер. 2025 р. 05:31 trymrock ***@***.***> пише:
  I'm having trouble selecting the frequency for the RTC6711. Can you help me?
RTC6711-Revice 2.4GHz.pdf
esp8266_test2.4G.zip—Reply to this email directly, view it on GitHub, or unsubscribe.You are receiving this because you commented.Message ID: ***@***.***>



trymrock left a comment (markohoepken/rx5808_pro_osd#7)
I'm having trouble selecting the frequency for the RTC6711. Can you help me?
RTC6711-Revice 2.4GHz.pdf
esp8266_test2.4G.zip
